package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@piwik/referrer-spam-blacklist",
+  "version": "0.0.0",
```

Comment:
Yea, the way I'm using this package right now is requiring it (it being the fork from which this commit comes from) [through Github](https://github.com/panstav/is-referrer-spammer/blob/master/package.json#L27) so Github releases are still relevant and the actual version on this package is meaningless since it's not on the npm registry, I believe the version field is required regardless of the source. 

Unless if you want it on npm - then the `0.0.0` should definitely be changed.